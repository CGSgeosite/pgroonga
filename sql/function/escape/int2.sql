SELECT pgroonga.escape(29::int2);
SELECT pgroonga.escape(-29::int2);
