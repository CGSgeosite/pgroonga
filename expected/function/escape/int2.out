SELECT pgroonga.escape(29::int2);
 escape 
--------
 29
(1 row)

SELECT pgroonga.escape(-29::int2);
 escape 
--------
 -29
(1 row)

